RunUnitTests.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
+dotnet restore --force Src/Support/GoogleApisClient.sln
```

Comment:
Do we need a restore first really?

I suspect it's worth doing a build of the whole solution too, in case there are projects that don't have explicit tests, but we still want to check for building.